pinecone/config/openapi.py

```diff
@@ -24,6 +24,8 @@ def build(cls, api_key: str, host: Optional[str] = None, **kwargs):
         openapi_config.host = host
         openapi_config.ssl_ca_cert = certifi.where()
         openapi_config.socket_options = cls._get_socket_options()
+        openapi_config.discard_unknown_keys = True
```

Comment:
Discovered this along the way. Seems like a better default behavior than erroring when unexpected data is returned.

Reply:
Looks sweet

Reply:
Good catch